test/parallel/test-inspect-support-for-node_options.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+const common = require('../common');
+const cluster = require('cluster');
+const assert = require('assert');
+
+common.skipIfInspectorDisabled();
+
+checkForInspectSupport('--inspect');
+
+function checkForInspectSupport(flag) {
+
+  const nodeOptions = JSON.stringify(flag);
+  process.env.NODE_OPTIONS = flag;
+
+  if (cluster.isMaster) {
+    for (let i = 0; i < 2; i++) {
+      cluster.fork();
+    }
+
+    cluster.on('online', (worker) => {
+      worker.disconnect();
+    });
+
+    cluster.on('exit', common.mustCall((worker, code, signal) => {
+      const errMsg = `For NODE_OPTIONS ${nodeOptions}, failed to start cluster`;
+      assert.strictEqual(worker.exitedAfterDisconnect, true, errMsg);
+    }, 2));
```

Comment:
Tiny nit: maybe make the `2` here and in the for loop that forks the workers a const since they should match?

Reply:
@richardlau I have added the const and replaced the `2` both in the for loop and callback.